src/module.c

```diff
@@ -6704,6 +6735,130 @@ int RM_SetModuleUserACL(RedisModuleUser *user, const char* acl) {
     return ACLSetUser(user->user, acl, -1);
 }
 
+/* Retrieve the user ID of the client connection behind the current context.
+ * The user ID can be used later, in order to check if command, key or channel
+ * can be executed or accessed according to the ACLs rules associated with that user.
+ * The caller can later free the user ID using the function RM_FreeUserID
+ *
+ * On success a RedisModuleUserID object is returned, otherwise
+ * NULL is returned. */
+ RedisModuleUserID *RM_GetCurrentUserID(RedisModuleCtx *ctx) {
```

Comment:
There is a random space here btw.
```suggestion
RedisModuleUserID *RM_GetCurrentUserID(RedisModuleCtx *ctx) {
```